aerospike-connection-manager/Makefile

```diff
@@ -24,7 +24,7 @@ build:
 	$(GOBUILD) .
 
 run: clean proto
-	$(GOBUILD) -o $(BINARY_NAME) -v .
+	$(GOBUILD) -v .
 	./$(BINARY_NAME) -config-file asld.toml
```

Comment:
Is the name of the default config file correct? Did we not change all the instances of `asld`?

Reply:
no, the config file and binary name remains asld.toml and asld 
maybe we can change the config file to acm-config.toml